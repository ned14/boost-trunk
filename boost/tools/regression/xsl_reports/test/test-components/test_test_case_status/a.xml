<?xml version="1.0" encoding="iso-8859-1"?>
<test-run source="CVS" runner="BoostConsulting" timestamp="2007-01-13T06:25:21Z" platform="Windows" tag="RC_1_34_0" run-type="incremental">
    <test-log status="expected" result="success" />
    <test-log status="expected" />
    <test-log status="expected" result="fail" />
    <test-log status="unexpected" result="success" />
    <test-log status="unexpected" result="fail" is-new="no" />
    <test-log status="unexpected" result="fail" toolset="old_compiler" library="unusable_library" />

</test-run>