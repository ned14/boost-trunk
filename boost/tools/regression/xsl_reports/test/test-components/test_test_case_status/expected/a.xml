<?xml version="1.0"?>
<results xmlns:exsl="http://exslt.org/common"><result>expected</result><result>expected</result><result>expected</result><result>success-unexpected</result><result>fail-unexpected</result><result>unusable</result></results>
